airbyte-integrations/connectors/source-twilio-singer/sample_files/catalog.json

```diff
@@ -0,0 +1,130 @@
+{
+  "streams": [
+    {
```

Comment:
```suggestion
    { "stream": {
```

Reply:
this data format has changed. This file should now have the `ConfiguredAirbyteCatalog` format. Suggestions apply the update.

Reply:
If I update this file as you suggest - I won't have the integration tests passing, I don't think it's okay?